cds/modules/opencast/api.py

```diff
@@ -230,11 +240,7 @@ def _ingest(self, session, media_package_xml, qualities):
                 form_data.update({dict_key: "false"})
 
         url = self.BASE_URL + "/ingest/cern-cds-videos"
-        current_app.logger.info(
-            "Opencast request for ingesting (qualities {0}): {1}".format(
-                qualities, url
-            )
-        )
+        self.log(url=url, opt="Qualilties: {0}".format(qualities))
```

Comment:
Can you please add a simple `console.logger.info` in `check_transcoding_status` when:
* there is nothing to do saying something like: `No started transcoding tasks to check, exiting`
* found tasks to check, add something like: `Triggering status check for x transcoding task`

This is because in the logs, we see a lot of `check_transcoding_status` but we don't really know if it will do something or not.